README.md

```diff
@@ -797,6 +797,82 @@ In either case:
     end
     ```
 
+### Nested conditionals
+
+* <a name="no-nested-conditionals"></a>
+  Avoid the use of nested conditionals for flow of control. 
+  ([More on this][avoid-else-return-early].) <sup>[[link](#no-nested-conditionals)]</sup>
```

Comment:
For bang methods, we also cite a blog: http://davidablack.net/dablog.html#2007/8/15/bang-methods-or-danger-will-rubyist 
I cited this one because it did a very good job articulating the rationale behind this. I think it's helpful for people to read it if they want a deeper dive. A second reason is I am copying some of the blog's wording and it seems unfair to do so without citing the author and giving them credit. 

Kent Beck's explanation is really nice too. I can link to that in addition, unless you have any objections?